tests/components/influxdb/test_init.py

```diff
@@ -144,7 +146,16 @@ async def test_setup_minimal_config(hass, mock_client, config_ext, get_write_api
     "mock_client, config_ext, get_write_api",
     [
         (influxdb.DEFAULT_API_VERSION, {"username": "user"}, _get_write_api_mock_v1),
-        (influxdb.DEFAULT_API_VERSION, {"token": "token"}, _get_write_api_mock_v1),
+        (
+            influxdb.DEFAULT_API_VERSION,
+            {"token": "token", "organization": "organization"},
+            _get_write_api_mock_v1,
+        ),
+        (
+            influxdb.API_VERSION_2,
+            {"api_version": influxdb.API_VERSION_2},
+            _get_write_api_mock_v2,
+        ),
```

Comment:
This test was tweaked because it was redundant. The intent when written was to test the logic in `validate_version_specific_config` that ensures if `api_version` is `"1"` then `token` and `organization` cannot be specified. But there is also a `vol.Inclusive` on `token` and `organization` so it was never getting to that point. And that code is already tested later with this test:
```
        (
            influxdb.API_VERSION_2,
            {"api_version": influxdb.API_VERSION_2, "organization": "organization"},
            _get_write_api_mock_v2,
        ),
```
So adjusted this one to test the version specific code by providing both an `organization` and a `token`.